app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,11 @@
+class LikesController < ApplicationController
+  def new; end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @like = @post.likes.new
+    @like.author_id = params[:user_id]
+
+    redirect_to user_post_path(params[:user_id], params[:post_id]) if @like.save
```

Comment:
- [ ] It will be nice to also tell the app to `redirect_to` somewhere in case the `@like.save` fails. You can refactor the entire `create` action to be similar to this:

```ruby
def create
   @post = Post.find(params[:post_id])
    @like = @current_user.likes.new(post_id: @post.id)

    respond_to do |format|
      if @like.save
        format.html { redirect_to @post }
      else
        format.html { redirect_to @post }
      end
    end
end
```